### ISSUE TITLE
aws_elb.health_check attributes does not update during update

### ISSUE THREAD

**Issue by gosuri** [hashicorp/terraform#915]
The `aws_elb` fails to update `health_check` attributes during update.  The below is the failure test case for `Terraform v0.3.7.dev (c18b01fa2a6214b084a2a95b8be765754b58198c)` trying to update the `healthy_threshold`. `healthy_threshold` remains at `5` and should be `10`.

The test output is at https://gist.github.com/gosuri/86a9027eab6394314273

```go
func TestAccAWSELBUpdate_HealthCheck(t *testing.T) {
	resource.Test(t, resource.TestCase{
		PreCheck:     func() { testAccPreCheck(t) },
		Providers:    testAccProviders,
		CheckDestroy: testAccCheckAWSELBDestroy,
		Steps: []resource.TestStep{
			resource.TestStep{
				Config: testAccAWSELBConfigHealthCheck,
				Check: resource.ComposeTestCheckFunc(
					resource.TestCheckResourceAttr(
						"aws_elb.bar", "health_check.3484319807.healthy_threshold", "5"),
				),
			},
			resource.TestStep{
				Config: testAccAWSELBConfigHealthCheck_update,
				Check: resource.ComposeTestCheckFunc(
					resource.TestCheckResourceAttr(
						"aws_elb.bar", "health_check.2648756019.healthy_threshold", "10"),
				),
			},
		},
	})
}

const testAccAWSELBConfigHealthCheck = `
resource "aws_elb" "bar" {
  name = "foobar-terraform-test"
  availability_zones = ["us-west-2a"]

  listener {
    instance_port = 8000
    instance_protocol = "http"
    lb_port = 80
    lb_protocol = "http"
  }

  health_check {
    healthy_threshold = 5
    unhealthy_threshold = 5
    target = "HTTP:8000/"
    interval = 60
    timeout = 30
  }
}
`
const testAccAWSELBConfigHealthCheck_update = `
resource "aws_elb" "bar" {
  name = "foobar-terraform-test"
  availability_zones = ["us-west-2a"]

  listener {
    instance_port = 8000
    instance_protocol = "http"
    lb_port = 80
    lb_protocol = "http"
  }

  health_check {
    healthy_threshold = 10
    unhealthy_threshold = 5
    target = "HTTP:8000/"
    interval = 60
    timeout = 30
  }
}
`
```

**Comment by pearkes** [comment]
I've successfully reproduced this. Tagged as bug, thanks!

**Comment by gosuri** [comment]
Fixed with pull request #916, closing this now. Thanks!
